packages/framework/src/Resources/elasticsearch/1.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
I don't know what is the proper name for the definition. It can be definition, settings, configuration,... But the point is that this is the only definition that elasticsearch have, so I don't see a reason for subdirectory.

What do You think?